tests/test_initialization.h

```diff
@@ -11,6 +11,9 @@
 #include "../src/nyx/environment.h"
 #include "../src/nyx/globals.h"
 
+using Nyxus::allocateTrivialRoisBuffers;
+using Nyxus::freeTrivialRoisBuffers;
```

Comment:
I don't think we need these. I removed them and compiled. The only think I had to add was `#include "../src/nyx/globals.h" ` in `test_main_nyxus.h`